### PR TITLE
ce_bgp_neighbor_af: fix a typo in module's parameter

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
ce_bgp_neighbor_af: fix a typo in module's parameter

##### ISSUE TYPE
- Docs Pull Request
- Bug Fix Pull Request

##### COMPONENT NAME
```lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py ```

### CODE REVIEW

**Comment by Andersson007** [comment]
@acozine , it is not about documentation only, it affects user interface.
That's a reason why i didn't include it to the typo-fixing PRs.
So i'm not sure who can / should merge this

File: lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py [hunk 1]
```diff
@@ -1095,8 +1095,8 @@ def check_bgp_neighbor_af_other(self, **kwargs):
                 else:
                     need_cfg = True
 
-        redirect_ip_vaildation = module.params['redirect_ip_vaildation']
-        if redirect_ip_vaildation != 'no_use':
+        redirect_ip_validation = module.params['redirect_ip_validation']
+        if redirect_ip_validation != 'no_use':
 
             conf_str = CE_GET_BGP_PEER_AF_HEADER % (
                 vrf_name, af_type, remote_address) + "<redirectIPVaildation></redirectIPVaildation>" + CE_GET_BGP_PEER_AF_TAIL
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: Thank you for your attention first.
But I don't think that is an issue.
Such as line 1102 which xml tag is 'redirectIPVaildation'.
So parameter 'redirect_ip_validation' is named from there.

Comment (Andersson007) [reply lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: do you suggest leaving the option name as it is?

Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: This is an interesting question.
I am sure that the word is from [NETCONF Schema API Reference](https://support.huawei.com/enterprise/en/doc/EDOC1100074782?idPath=24030814%7C21782165%7C21782239%7C22318639%7C21059820).

Comment (Andersson007) [reply lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: i can't open that. it is a clear typo anyway, looks strange, better to inform huawei about this.
i added the aliase, is it ok?

Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: That sounds good!

**Comment by xuxiaowei0512** [comment]
+1 
The aliase will not make a difference to existing playbooks.

File: lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py (hunk 1 above)
Comment (Andersson007) [reply lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py:1103]: @xuxiaowei0512 if you agree, would be cool if you approved this :) thanks

**Review by yuandongx — APPROVED** [review]
+1

**Review by yuandongxx — APPROVED** [review]
+1

**Comment by hwDCN** [comment]
@Andersson007  thank you so much.

**Comment by Andersson007** [comment]
@xuxiaowei0512 , @hwDCN thank you for reviewing and approving this!

**Comment by gundalow** [comment]
@Andersson007 @hwDCN @xuxiaowei0512 Thank you.

**Comment by Andersson007** [comment]
@gundalow , thanks for merging!

**Pull request merged**